pkg/render/manifests.go

```diff
@@ -62,6 +62,7 @@ func (c *clusterManifestContext) setupManifests(externalOauth, includeRegistry b
 	if includeRegistry {
 		c.registry()
 	}
+	c.roksMetrics()
```

Comment:
To go along with my comment above, we'll either need an `if ROKSMetricsImage != ""` here, or hold off on merging this until our code is prepared to pass the image over.

Reply:
hmm, you're right, I think we should make this optional until the image is specified. Will change.